payments-core/src/main/java/com/stripe/android/model/PaymentMethodCreateParams.kt

```diff
@@ -40,6 +40,11 @@ data class PaymentMethodCreateParams internal constructor(
     /**
      * If provided, will be used as the representation of this object when calling the Stripe API,
      * instead of generating the map from its content.
+     *
+     * The map should be valid according to the
+     * [PaymentMethod creation API](https://stripe.com/docs/api/payment_methods/create)
+     * documentation, including a required `type` entry.
+     *
      * The values of the map must be any of the types supported by [android.os.Parcel.writeValue].
      */
     private val overrideParamMap: Map<String, @RawValue Any>? = null,
```

Comment:
This could cause parcelization issues. I've run into issues with `@RawValue`.

Reply:
Only if the type is not supported by `android.os.Parcel.writeValue`, right? We can't enforce that, but that's documented.

Reply:
Yeah, I think you're right. I believe I ran into issues with `null` values.